.github/workflows/aks-auto-shutdown.yaml

```diff
@@ -0,0 +1,14 @@
+name: AKS Auto Shutdown
+
+on:
+  workflow_dispatch:
+  schedule:
+    # Run everyday
+    # - cron: '0 20 * * *' # Everyday at 8pm UTC (Will enable after tested)
```

Comment:
Looks good, assuming this will be enabled in a subsequent PR?

Reply:
Yep! Just didn't want the schedule running if it hasn't been tested yet :)